src/styles/blocks/catalog.scss

```diff
@@ -0,0 +1,20 @@
+@import "./card.scss";
+@import "./star.scss";
+
+.cards {
+  padding: 50px 0;
+  max-width: 944px;
+  margin: 0 auto;
+
+  &__row {
+    display: flex;
+
+    > :not(:nth-child(4n)) {
+      margin-right: 48px;
+    }
+  }
```

Comment:
i'd rather use `gap`, but not such a difficult selector 😉

Reply:
![image](https://user-images.githubusercontent.com/65117662/173028990-50a6d512-b1ed-4256-80bd-e277ea2a8959.png)